doc/release-notes-1.13.rst

```diff
@@ -21,14 +21,18 @@ Kernel
 Architectures
 *************
 
+* nxp_imx: Added RDC (Resource Domain Controller) configuration for i.MX SoCs to share peripherals between the Cortex A and Cortex M4 cores.
```

Comment:
Please keep line length < 80 chars please.  Also your commit message has a line that's too long (see https://app.shippable.com/github/zephyrproject-rtos/zephyr/runs/21594/5/tests)